packages/driver/cypress/integration/e2e/multi-domain/multi_domain_yield_spec.ts

```diff
@@ -118,7 +118,7 @@ describe('cy.origin yields', () => {
       done()
     })
 
-    cy.origin('http://foobar.com:3500', () => {
+    cy.origin<JQuery>('http://foobar.com:3500', () => {
```

Comment:
Should we be doing something different with types to avoid casting?

Reply:
No, I think this is good way to handle the yielded value type. If you don't actually use the yielded value, its type doesn't matter, so you don't need to tell TypeScript what the type is. If you do use the value, it's beneficial to make it clear what the type is instead of it being `any`.